rmom/src/broadcast/stream_server.rs

```diff
@@ -91,13 +93,50 @@ impl MessageStream for StreamServer {
         &self,
         request: Request<CreateQueueRequest>,
     ) -> Result<Response<CreateQueueOkResponse>, Status> {
-        // TODO: Finish off implementation
-        unimplemented!()
+        let label = request.into_inner().queue_label;
+        info!("Request to CREATE queue with label: {}", label);
+
+        let mut be_map = self.broadcast_ends.lock().unwrap();
```

Comment:
It is never a good idea to name variables after a data structure. What if in the future we decide not to use a map?
Also be_map is a very confusing name.